src/config/dynamic-theme-fixes.config

```diff
@@ -8747,6 +8747,16 @@ CSS
 
 ================================
 
+nmap.org
+seclists.org
+
+CSS
+img[src$="topleftcurve.gif"] { 
+    display: none; !important
```

Comment:
```suggestion
    display: none !important;
```

Reply:
@Gusted Sorry, I don't see a button to accept these changes (this and the nmap commit) unlike the others for some reason. If you could just push them through on your end that would be appreciated.

Reply:
It seems like you've disabled the maintainers ability to edit the PR, see https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
My bad, I think it's because I nuked my forks after submitting these patches. I'll try closing these two pull requests and send in new ones.